pkg/apis/core/v1/storagecluster.go

```diff
@@ -199,9 +199,10 @@ type CSITopologySpec struct {
 
 // SecuritySpec is used to define the security configuration for a cluster.
 type SecuritySpec struct {
-	Enabled bool      `json:"enabled,omitempty"`
-	Auth    *AuthSpec `json:"auth,omitempty"`
-	TLS     *TLSSpec  `json:"tls,omitempty"`
+	Enabled    bool      `json:"enabled,omitempty"` // CHECKME: remove this flag?
+	Privileged bool      `json:"privileged,omitempty"`
```

Comment:
Let's not add a first class spec parameter to enable a back door to go back to the previous method. 

Okay to keep a new annotation so achieve the same effect.

Reply:
Just to check if I understood correctly -- you'd want to move the `Security: Privileged` tag into the annotation?

Note that this is not a backdoor -- I picked this method since it is the "closest" to how people normally enable the privileged PODs, which is...
```
spec:
   containers:
      securityContext:
         privileged: true
```